modules/core/src/persistence_base64_encoding.cpp

```diff
@@ -333,6 +334,10 @@ size_t base64::RawDataToBinaryConvertor::make_to_binary_funcs(const std::string
                     size = sizeof(uchar);
                     pack.func = to_binary<uchar>;
                     break;
+                case 'b':
+                    size = sizeof(bool);
```

Comment:
sizeof(bool) may be different on different platforms. `to_binary()` converts it to uchar.

Reply:
Done